tests/cases/standalone/common/tql/basic.result

```diff
@@ -59,6 +59,28 @@ TQL EVAL (0, 10, '5s') test{k="a"};
 | 2.0 | 1970-01-01T00:00:10 | a |
 +-----+---------------------+---+
 
+TQL EVAL ('1970-01-01T00:00:00'::timestamp, '1970-01-01T00:00:00'::timestamp + '10 seconds'::interval, '1s') test{k="a"};
+
++-----+---------------------+---+
+| i   | j                   | k |
++-----+---------------------+---+
+| 2.0 | 1970-01-01T00:00:01 | a |
+| 2.0 | 1970-01-01T00:00:02 | a |
+| 2.0 | 1970-01-01T00:00:03 | a |
+| 2.0 | 1970-01-01T00:00:04 | a |
+| 2.0 | 1970-01-01T00:00:05 | a |
+| 2.0 | 1970-01-01T00:00:06 | a |
+| 2.0 | 1970-01-01T00:00:07 | a |
+| 2.0 | 1970-01-01T00:00:08 | a |
+| 2.0 | 1970-01-01T00:00:09 | a |
+| 2.0 | 1970-01-01T00:00:10 | a |
++-----+---------------------+---+
+
+TQL EVAL (now() - '2 minutes'::interval, now(), '5s') test{k="a"};
+
```

Comment:
frankly speaking, I don't fully understand this result. I suspect that it's related to `test` table initialisation.

Reply:
This means the result set is empty. `test` contains data around 1970, so querying `now()` won't have a result. You can try to change this to something like `now() - now() - '2 minutes'::interval'`

Reply:
actually, that's an interesting one,
At the moment current implementation doesn't allow it and fail with 
```
ParserError("Failed to simplify expression ArrowError(InvalidArgumentError(\"Invalid arithmetic operation: Duration(Nanosecond) + Interval(MonthDayNano)\"))")
```
I curious do we need to support it?